crates/accelerate/src/sabre/heuristic.rs

```diff
@@ -165,6 +165,45 @@ impl DecayHeuristic {
     }
 }
 
+/// Define the characteristics of the "depth" heuristic. This is used to penalize swaps that increase the circuit depth.
+#[pyclass]
+#[pyo3(module = "qiskit._accelerate.sabre", frozen)]
+#[derive(Clone, Copy, PartialEq)]
+pub struct DepthHeuristic {
+    /// The relative weighting of this heuristic to others.  Typically you should just set this to
+    /// 1.0 and define everything else in terms of this.
+    pub weight: f64,
```

Comment:
Yes, you are correct, the weight should not be 1. I've adjusted it so that the weight is now less than 1, relative to the 'basic' heuristic. Specifically, if the basic heuristic has a weight of 1, the depth heuristic should default to 1/3. This reflects that a score of 3 in the depth heuristic corresponds to a depth increase of 3, which is equivalent to 1 swap (or 3 CNOT gates), corresponding to a distance of 1.